### PR TITLE
[UI]: Page grid and adaptive layout (without card)

### PR DESCRIPTION
**Pull request by DenysDubas**

### CODE REVIEW

File: src/componentsOfCartExample/productsCart/productsCart.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  initialCount?: number;
+};
+
+export const Product: React.FC = ({ initialCount = 1 }: CounterProps) => {
+  const [count, setCount] = useState(initialCount);
+  const [quantity, setQuantity] = useState(1);
+
+  const increment = () => {
+    setCount(count + 1);
+    setQuantity(count + 1);
```
Comment (KuharchukDmytro) [inline src/componentsOfCartExample/productsCart/productsCart.tsx:17]:
```suggestion
    setCount(prev => prev + 1);
    setQuantity(prev => prev + 1);
```

File: src/componentsOfCartExample/productsCart/productsCart.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IconButton } from '@material-ui/core';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Delete } from '@material-ui/icons';
```
Comment (KuharchukDmytro) [inline src/componentsOfCartExample/productsCart/productsCart.tsx:5]: add package,json

File: src/componentsOfCartExample/cartsCheckout/checkout.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
+h1,
+h2,
+p {
+    margin: 0;
+}
```
Comment (KuharchukDmytro) [inline src/componentsOfCartExample/cartsCheckout/checkout.scss:5]: remove

File: src/componentsOfCartExample/cartsCard/card.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Product } from '../productsCart/productsCart';
+import { Checkout } from '../cartsCheckout/checkout';
+// import '../../App.scss';
```
Comment (KuharchukDmytro) [inline src/componentsOfCartExample/cartsCard/card.tsx:3]: remove

File: src/App.tsx [hunk 5]
```diff
@@ -1,5 +1,9 @@
 import './App.scss';
 
+// App.tsx
```
Comment (KuharchukDmytro) [inline src/App.tsx:3]: remove

File: src/container/container.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
```
Comment (KuharchukDmytro) [inline src/container/container.scss:9]: remove

File: src/container/container.scss [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    max-width: 1200px;
+    margin: 0 32px;
+    padding: 0;
+  }
+  .card {
+    flex-basis: 25%;
+  }
+}
+
+@media only screen and (min-width: 768px) and (max-width: 1198px) {
```
Comment (KuharchukDmytro) [inline src/container/container.scss:43]: use mixins

**Review by KuharchukDmytro — CHANGES_REQUESTED** [review]

**Review by lilia-obushenko — APPROVED** [review]

**Review by KuharchukDmytro — APPROVED** [review]

**Pull request merged**